README.md

```diff
@@ -11,78 +11,31 @@
-
-### Update the content in the readme file
-
-After you implement the logic for your module and create examples and tests, update this readme file in your repository by following these steps:
-
-1.  Update the title heading and add a description about your module.
-1.  Update the badge links.
-1.  Remove all the content in this H2 heading section.
-1.  Complete the [Usage](#usage) and [Required IAM access policies](#required-iam-access-policies) sections. The [Examples](#examples) and [Requirements](#requirements) section are populated by a pre-commit hook.
-
-### Commit your code and submit your module for review
-
-1.  Before you commit any code, review [Contributing to the IBM Cloud Terraform modules project](https://terraform-ibm-modules.github.io/documentation/#/contribute-module) in the project documentation.
-1.  Create a pull request for review.
-
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
-
-1.  Create a PR to enable the upgrade test by removing the `t.Skip` line in `tests/pr_test.go`.
-
+## Summary
+
+This repository contains wazi deployable architecture solutions that help provision VPC landing zones and interconnect them. The below solutions are available in the IBM Cloud Catalog and can also be deployed with terraform.
+
+Two solutions are offered:
+1. [Quickstart Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/standard_quickstart/solutions/quickstart)
+    This pattern deploys the following infrastructure:
+    - Increases security with Key Management.
+    - Collects and stores Internet Protocol (IP) traffic information with Activity Tracker and Flow Logs.
+    - Simplifies risk management and demonstrates regulatory compliance with Financial Services
+    - Uses an edge VPC for secure access through the public internet.
+2. [Standard Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/standard_quickstart/solutions/standard)
+    This pattern deploys the following infrastructure:
+    - Seperate VPC for edge.
+    - Seperate VPC for Workloads.
+    - Virtual Server Instances for every subnet.
+    - Increases security with Key Management.
+    - Collects and stores Internet Protocol (IP) traffic information with Activity Tracker and Flow Logs.
+    - Securely connects to multiple networks with a client-to-site/site-to-site virtual private network
+    - Simplifies risk management and demonstrates regulatory compliance with Financial Services
+    - Uses an edge VPC for secure access through the public internet.
 <!-- Remove the content in this previous H2 heading -->
 ## Reference architectures
+- [Quickstart Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/standard_quickstart/solutions/quickstart)
+- [Standard Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/standard_quickstart/solutions/standard)
```

Comment:
The links for Quickstart and Standards variations are pointing to the standard_quickstart branch but will eventually need to be updated to point to the main branch.

Reply:
Made changes as suggested after it will get merge into the main branch the link can be accessible.